apps/common-app/src/apps/reanimated/examples/DynamicColorIOSExample.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useEffect } from 'react';
+import {
+  DynamicColorIOS as DynamicColorIOSNative,
```

Comment:
```suggestion
  DynamicColorIOS as RNDynamicColorIOS,
```

Maybe change this or just use our `DynamicColorIOS` implementation everywhere instead?